dev/AppLifecycle/AppInstance.cpp

```diff
@@ -0,0 +1,415 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include <pch.h>
+#include <AppInstance.h>
+#include <Microsoft.Windows.AppLifecycle.AppInstance.g.cpp>
+
+#include "ActivationRegistrationManager.h"
+#include "LaunchActivatedEventArgs.h"
+#include "ProtocolActivatedEventArgs.h"
+#include "FileActivatedEventArgs.h"
+#include "Association.h"
+#include "ExtensionContract.h"
+
+using namespace winrt;
```

Comment:
isn't this asking for trouble with respect to calling to the ABI instead of the projection?